documentation/src/docs/wire-protocol.md

```diff
@@ -102,3 +139,59 @@ After receiving the "Append Setup" reply, it performs the following:
 While this is happening, the server will be periodically sending it `DataAppended` replies acking messages. Note that there can be multiple "Appends Setup" for a given TCP connection. This allows a client to share a connection when producing to multiple Segments.
 
 A client can optimize its appending by specifying a large value in it's `AppendBlock` message, as the events inside of the block do not need to be processed individually.
+
+## Segment Attribute API
+
+The following Paramters will be used by all the below mentioned APIs.
```

Comment:
@shrids @tkaitchuck  This statement was included to state that these parameters will be used  in common.